### PR TITLE
created rough pubchemchem_sources.ttl

### PR DESCRIPTION
**Pull request by jharri43**
I'm not sure how to to initialize the objects of the rdf. 
Can I just make anything up as a subject and then make it a subClassOf something as I have done in order to create the desired hierarchy?
Also, can subjects be strings with blank spaces?

@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
@prefix obo:   <http://purl.obolibrary.org/obo/> .
@prefix src:   <https://pubchem.ncbi.nlm.nih.gov/source/> .

rdfs:Source .

rdfs:PubChem_Source rdfs:subClassOf Source .

src:Broad Institute
  rdfs:label "Broad Institute" ;
  rdfs:subClassOf Source ;
  rdfs:type PubChem_Source .
  obo:IAO_0000115 "A PubChem Bioasay source."

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -17,11 +17,6 @@
 	<classpathentry exported="true" kind="lib" path="lib/xercesImpl-2.11.0.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/xml-apis-1.4.01.jar"/>
 	<classpathentry kind="con" path="org.eclipse.jst.j2ee.internal.module.container"/>
-	<classpathentry kind="con" path="org.eclipse.jst.server.core.container/org.eclipse.jst.server.tomcat.runtimeTarget/Apache Tomcat v8.0">
```
Comment (aclarkxyz) [inline .classpath]: Unstage this file from Git: it'll break the rest of the project.

File: .settings/org.eclipse.wst.common.project.facet.core.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (aclarkxyz) [inline .settings/org.eclipse.wst.common.project.facet.core.xml]: Ditto: this file needs to be rolled back. In future don't add files unless you know they're supposed to be different.

Comment (gedeck) [reply .settings/org.eclipse.wst.common.project.facet.core.xml]: I have a `~/.gitignore_global` file that contains files I in general want to ignore. They can always be added if needed. The most important ones are the . files.
```
patchLabels.py
.externalToolBuilders/
/rdkit/Chem/inchi.py
/.project
/.cproject
/.classpath
/.texlipse
/.pydevproject
*.so
*.swp
*.pyc
/.pydevproject/
/lib/*
.DS_Store
/.settings/*
/Debug/
/src/log4j2.xml
/.scannerwork
/sonar-project.properties
/jacoco.exec
/baeconsole.pg
```

Comment (aclarkxyz) [reply .settings/org.eclipse.wst.common.project.facet.core.xml]: I thought the .gitignore mechanism had no effect on files that are already staged? Things like .classpath actually need to be in the project, but they need to not be modified gratuitously if you had to tweak it to match your own unique configuration...

Comment (gedeck) [reply .settings/org.eclipse.wst.common.project.facet.core.xml]: That is the case. It's more relevant for the .settings directory and some other stuff that I in general want to exclude.

**Pull request merged**